### ISSUE TITLE
Pico 2 support?

### ISSUE THREAD

**Issue by Dexus** [phil-barrett/PicoCNC#30]
Would this word support the pico2? Or only the pico1?

**Comment by phil-barrett** [comment]
Yes, the Pico2 fits in the board.  Currently, there is not a version of grblHAL that supports it but I have heard that it may be coming.

Phil

**Comment by phil-barrett** [comment]
We are making progress.  The Web Builder now supports building grblHAL for a Pico2 (RP2350) plugged into a PicoCNC board.  There are a couple of issues with timing for step generation that I think will be fixed soon. 

![image](https://github.com/user-attachments/assets/67bd2c9d-dcd7-46f9-83cf-dcdc01003e22)

**Comment by Dexus** [comment]
Have seen it. Now I await my order be shipped ;) from US to DE. Thank you.

one more thing I would like to know it should be easy to use a diode laser on that board right? Or should I know something special?

Best Regards josef

**Comment by phil-barrett** [comment]
[This might help](https://www.grbl.org/single-post/using-a-diode-laser-with-grblhal)

**Comment by Dexus** [comment]
@phil-barrett only to be sure, i have to switch the 0 resistor from R40 to R39 right for laser mode?

**Comment by phil-barrett** [comment]
Resolved. (it works just fine now)
